<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Research Interests</title>

<script src="site_libs/header-attrs-2.22/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>






<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Jiatong Liang</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About Me</a>
</li>
<li>
  <a href="research.html">Research</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="photos\resume1.pdf">CV</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Research Interests</h1>

</div>


<!--- <h3>Research Interests</h3> --->
<ul>
<li>
Statistical machine learning algorithms for precision medicine and
computational Biology with a particular emphasis on Computational
Pathology
</li>
<li>
Deep learning algorithms for large scale medical imaging
(e.g. <a href="https://www.pixelscientia.com/article-finding-prognostic-patterns-in-gigapixel-images.html">whole
slide images</a>)
</li>
<li>
Computational/statistical approaches for tissue omics
e.g. characterization of the tumor microenvironment through H&amp;E,
multiplex immunofluorescence, spatial transcriptomics, etc
</li>
<li>
Multi-modal/view data analytic methodology
</li>
<li>
Statistical analysis of structured data (networks, structured sparsity,
etc)
</li>
</ul>
<hr>
<p><br> <!---
<h3>Research Interests</h3>
<ul>
    <li> Statistical machine learning </li>
    <li> Precision medicine and compuational pathology</li>
    <li> Deep learning for medical image analysis </li>
    <li> Structured sparsity </li>
    <li> Network analysis </li>
    <li> Multi-view/modal data integration </li>
    <li> Structured sparsity/high dimensional regression </li>
</ul>
<hr><br>
---></p>
<h3>
Conference Presentations and Posters
</h3>
<ul>
<li>
Carmichael, I., Song, A.H., Chen, R.J., Williamson, D.F.K., Chen, T.Y.,
Mahmood, F. Incorporating intratumoral heterogeneity into
weakly-supervised deep learning models via variance pooling. Digital
Pathology &amp; AI Congress USA. New York, NY. June, 2022.
</li>
<br>
<li>
Olayinka, O., Carmichael, I., Gaeta Gazzola, M., Dimeola, K.A., Zheng,
Z., Madden, L.M., Beitel, M., Barry, D.T. Sex Differences among
Individuals with Chronic Pain and Opioid Use Disorder Entering Methadone
Maintenance Treatment. College on Problems of Drug Dependence.
Minneapolis, Minnesota. June, 2022.
</li>
<br>
<li>
Gaeta, M., Carmichael, I., Madden, L., Beitel, M., Eggert, K., Barry,
D.T. Patient Characteristics and Retention among Homeless Patients
Enrolled in a Low-Barrier-To Treatment-Entry Medication for Opioid Use
Disorder Program. College on Problems of Drug Dependence. Online. June,
2021.
</li>
<br>
<li>
Fusion of image and genetic data with convolutional neural networks and
AJIVE, <a href="">Bayesian, Fiducial, and Frequentist (BFF)
Conference</a>. Durham, NC. April, 2019.
(<a href="/assets/carmichael_bff_2019_compressed.pdf">poster</a>)
</li>
<br>
<li>
Angle-based Joint and Individual Variation Explained with Applications
to Image and Genetic Data,
<a href="https://www.mn.uio.no/math/english/research/projects/focustat/news-and-events/">FocuStat
Combo Kitchen</a>. Oslo, Norway. November, 2018
</li>
<br>
<li>
Anlge Based Joint and Individual Variation Explained,
<a href="https://www.bi.vt.edu/nsf-big-data-2018"> NSF Big Data Hubs and
Spokes Join PI meeting</a>. Alexandria, VA. June, 2018.
(<a href="/assets/ajive_carmichael_nsf_bigdata2018_poster.pdf">poster</a>)
</li>
</ul>
<hr>
<p><br></p>
<h3>
Talks
</h3>
Slides not linked to below are available upon request. <br>
<ul>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. UC Berkeely,
Department of Biostatistics. Berkeley, CA. March, 2022.
</li>
<br>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. The Flatiron
Institute, Center for Computational Mathematics. March, 2022.
</li>
<br>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. University of British
Columbia, Department of Statistics. Vancouver, Canada. Feb, 2022.
</li>
<br>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. University of
Michigan, Department of Biostatistics. Ann Arbor, MI. Jan, 2022.
</li>
<br>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. NYU Grossman School
of Medicine, Division of Biostatistics. NYC, NY. November, 2021.
</li>
<br>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. University of North
Carolina at Chapel Hill, Department of Statistics and Operations
Research. Chapel Hill, NC. September, 2021.
</li>
<br>
<li>
The folded concave Laplacian spectral penalty learns block diagonal
sparsity patterns with the strong oracle property. University of
Washington, Department of Statistics. Seattle, WA. June, 2021.
</li>
<br>
<li>
Sparsity Structure Estimation for Multi-View Mixture Models. University
of Washington, Department of Statistics. Seattle, WA. May, 2020.
</li>
<br>
<li>
Joint and individual analysis of breast cancer histologic images and
genomic covariates, Harvard Medical School. Boston, MA. December, 2019.
(<a href="/assets/joint_image_genetics.pdf">Slides</a>)
</li>
<br>
<li>
Joint and individual analysis of histopathology images and genetic
covariates. Computational Medicine group, UNC Chapel Hill. May, 2019.
</li>
<br>
<li>
Angle-based Joint and Individual Variation Explained with Applications
to Image and Genetic Data. University of Illinois Urbana-Champaign,
Department of Statistics. Urbana, IL. February, 2019.
</li>
<br>
<li>
Angle-based Joint and Individual Variation Explained with Applications
to Image and Genetic Data. University of Wisconsin-Madison, Department
of Statistics. Madison, WI. January, 2019.
</li>
<br>
<li>
Angle-based Joint and Individual Variation Explained with Applications
to Image and Genetic Data. Harvard University, Department of
Biostatistics. Boston, MA. January, 2019.
</li>
<br>
<li>
Joint analysis of H&amp;E stained images and genetic covariates using
deep learning and AJIVE. GenStat group, UNC Chapel Hill. September,
2018.
</li>
<br>
<li>
<a href="http://stat-or.unc.edu/event/stor-colloquium-iain-davis-unc-chapel-hill">
Data Science and the Undergraduate Curriculum</a>. UNC STOR Department
Colloquium. Chapel Hill, NC. October, 2017.
(<a href="/assets/data_science_stor_colloquium.pdf">Slides</a>)
</li>
<br>
<li>
<a href="https://www.youtube.com/watch?v=AP7_godzwVI"> Open Data,
Networks and the Law</a> by Iain Carmichael and Michael Kim. PyData
Carolinas, 2016.
</li>
</ul>
<hr>
<br>
<h3>
Past Projects
</h3>
<ul>
<br>
<li>
<b>Probabilistic Programming</b> (summer 2016)<br> Summer internship at
<a href="https://gamalon.com/">Gamalon Machine Intelligence</a>. Worked
on theoretical problems in probabilistic programming and contributed to
the development of Chimpy, a probabilistic programming language in
Python.
</li>
<br>
<li>
<b>Regional Coverage from Space Systems</b> (summer 2012)<br> Developed
algorithms for satellite regional coverage problems at the NSF sponsored
<a href="http://www.ipam.ucla.edu/programs/student-research-programs/research-in-industrial-projects-for-students-rips-2017/">RIPS</a>
program. Collaboration with
<a href="http://www.ipam.ucla.edu/">Institute for Pure and Applied
Mathematics</a> and the <a href="http://www.aerospace.org/">Aerospace
Corporation</a>.
</li>
</ul>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
